pkgs/applications/graphics/xv/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub,
+  cmake, xorg, libpng, libwebp, libtiff, libjpeg, jasper,
+  }:
```

Comment:
To minimize the diff on dependency renames and additions the list is usually one attribute per line. Otherwise LGTM.